pyqtlet2/leaflet/layer/marker/marker.py

```diff
@@ -28,7 +28,7 @@ def _onMove(self, event):
         self._logger.debug('marker moved. event: {event}'.format(event=event))
         event = self._qJsonValueToDict(event)
         self.latLng = [event["latlng"]["lat"], event["latlng"]["lng"]]
-        self.move.emit(event)
+        self.move.emit({**self._qJsonValueToDict(event), "latLng": self.latLng, "sender": self})
```

Comment:
The `latLng` values are already contained in the event, to reduce redundent values, I would suggest to leave the `latLng` field out. 
```suggestion
        self.move.emit({**self._qJsonValueToDict(event), "sender": self})
```